Bypass Recaptcha for automated testing

mythilytm opened the pull request:
## Description
- Added `data-bypassCaptcha` as a html script attribute to help bypass recaptcha for uptime monitoring
- Sends e2eTestMode attribute with message

This has been tested in aselo development.

### Checklist
- [ ] Corresponding issue has been opened
- [ ] New tests added

<!-- Uncomment for web client-related PRs
- [ ] English strings reviewed and copyedited
- [ ] Strings are localized
- [ ] Images are compressed (TinyPNG or svgo)
- [ ] Sample environment file updated (when environment variables changed)
- [ ] Verified on mobile
- [ ] Verified on desktop
- [ ] Verified accessibility ([Accessibility - Web applications](https://docs.google.com/document/d/1VwPDyLqw7r_iISMgDZr1FdAmAqqOIIVIxQqsYA-HaHE/edit?usp=sharing))
-->

### Related Issues
Fixes # [CHI-2047](https://tech-matters.atlassian.net/browse/CHI-2047)

### Verification steps
- Check locally that reCaptcha is bypassed
- This branch has been deployed to E2E webchat -  https://tl-public-chat-e2e-dev.s3.amazonaws.com/e2e-chat-development.html. Check that the script for html has the `bypasscaptcha` flag as a data attribute

[CHI-2047]: https://tech-matters.atlassian.net/browse/CHI-2047?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

robert-bo-davis reviewed (approved):
I've added a few changes to this PR, so I also added @mythilytm as a reviewer... I don't think you can actually review your own PR, but you can give an LGTM once you review.

mythilytm commented on configurations/e2e-development.ts:
```diff
@@ -22,6 +22,7 @@ const flexFlowSid = 'FOfce25fd1dff726dcdae2899de86de6c5';
 const defaultLanguage = 'en-US';
 const captureIp = true;
 const contactType: ContactType = 'ip';
+const enableRecaptcha = false;
```
> const enableRecaptcha = false;
Ah, good catch! I overlooked this detail.

mythilytm commented:
> I've added a few changes to this PR, so I also added @mythilytm as a reviewer... I don't think you can actually review your own PR, but you can give an LGTM once you review.
LGTM
